src/data/detail-page/resources/ResourcesTypeNames.ts

```diff
@@ -0,0 +1,12 @@
+import { ResourcesType } from './ResourcesInterfaces';
+
+export const ResourcesTypeNames: Record<ResourcesType, string> = {
```

Comment:
I decided to use `Record` instead of `enum` because I also use `ResourceType` in `Resources` interface and I wanted to keep consistency between them. Is `enum` better here or it is a matter of choice?

Reply:
enums are the go-to solution when we want a predefined list of values, however, you're right, that in this case there is also the issue of keeping the types consistent, so Record is also a good option :)
Just a fun fact for future reference: we could create a type using enum keys (but I'm not sure if this would be more readable than the ResourcesType type that we have now):
```
export enum ResourcesTypeNames {
  books = 'Books',
  courses = 'Online courses',
  platforms = 'Platforms',
  tools = 'Useful tools',
  communities = 'Communities',
  portalsAndPodcasts = 'Portals and Podcasts',
  games = 'Games & Coding Challenges',
  varia = 'Varia'
}

export type ResourcesType = keyof typeof ResourcesTypeNames; // this could replace our ResourcesType type

Lastly, maybe we could rename `ResourcesType` and make it singular (that is `ResourceType`)? Plurals in the names describing singular entities tend to be misleading.

Reply:
`export type ResourcesType = keyof typeof ResourcesTypeNames` is great for this problem :) I changed the code according to it and I added `enum` instead of `Record`.